services/blockstorage/service.go

```diff
@@ -83,6 +70,17 @@ func NewBlockStorage(ctx context.Context, config config.BlockStorageConfig, pers
 	return s
 }
 
+func (s *service) GetLastCommittedBlockHeight(ctx context.Context, input *services.GetLastCommittedBlockHeightInput) (*services.GetLastCommittedBlockHeightOutput, error) {
```

Comment:
still not convinced that this should return the timestamp as well - in all except one of the usages we only care about the hight and not the timestamp - in that place we can get the entire block to solve it and simplify boxing and unboxing in others